src/lib/dnssd/platform/Dnssd.h

```diff
@@ -230,8 +230,10 @@ CHIP_ERROR ChipDnssdBrowse(const char * type, DnssdServiceProtocol protocol, chi
  *
  * @param browseIdentifier an identifier for a browse operation that came from
  *                         ChipDnssdBrowse.
+ * @param callback         The callback to call when the browse is stopped.
+ *
  */
-CHIP_ERROR ChipDnssdStopBrowse(intptr_t browseIdentifier);
+CHIP_ERROR ChipDnssdStopBrowse(intptr_t browseIdentifier, DnssdBrowseCallback callback = {});
```

Comment:
This doesn't make sense.   The API documentation is pretty clear: the DnssdBrowseCallback that gets called is the one that was passed to ChipDnssdBrowse.  What is the point of this second DnssdBrowseCallback?

Also note that doing this break all the other platforms...

Reply:
Right. This is causing CI failures in the other platforms... I thought it would be treated as an optional parameter. 
connectedhomeip/src/platform/android/java/chip/platform/NsdManagerServiceBrowser.java  stopDiscover() was expecting this second DnssdBrowseCallback but it can be retained in stopDiscover() instead of being passed around. Addressed in latest commit.